File: TaskManagementApp/src/app/models/task.ts
export class Task
{
    id: number;
    title: string;
    description: string;
    createdDate: string;
    dueDate: string;
    isCompleted: boolean;
    user: string;
    constructor(id: number, title: string, description: string, createdDate: string, dueDate: string, isCompleted: boolean, user: string)
    {
        this.id = id;
        this.title = title;
        this.description = description;
        this.createdDate = createdDate;
        this.dueDate = dueDate;
        this.isCompleted = isCompleted;
        this.user = user;
    }
}

File: TaskManagementApi/Models/Task.cs
namespace TaskManagementApi.Models
{
    public class Task
    {
        public int Id { get; set; }
        public string Title { get; set; } = "";
        public string Description { get; set; } = "";
        public string CreatedDate { get; set; }
        public string DueDate { get; set; }
        public bool IsCompleted { get; set; }

        public string User { get; set; } = "";
    }

}
